apps/ledger-live-mobile/src/screens/Account/ListHeaderComponent.tsx

```diff
@@ -130,6 +136,15 @@ export function getListHeaderComponents({
 
   const stickyHeaderIndices = empty ? [] : [0];
 
+  const [latestOperation] = groupAccountOperationsByDay(account, {
+    count: 2,
+  }).sections[0].data.filter(operation => operation.type === "OUT");
+
+  const shouldRenderEditTxModal =
+    mainAccount.currency.family === "ethereum" &&
+    latestOperation?.blockHeight === null;
```

Comment:
Checks if the transaction is still pending or not

Reply:
Same thing as above, I think it could be covered by the `isEditableOperation` helper of Huiqi, let's discuss this with him.